ajax/libs/web3/package.json

```diff
@@ -0,0 +1,52 @@
+{
+  "name": "web3",
+  "version": "0.19.0",
+  "description": "Ethereum JavaScript API",
+  "filename": "web3.min.js",
+  "license": "LGPL-3.0",
+  "repository": {
+    "type": "git",
+    "url": "https://github.com/ethereum/web3.js.git"
+  },
+  "homepage": "https://github.com/ethereum/wiki/wiki/JavaScript-API",
+  "keywords": [
+    "Ethereum",
+    "JavaScript",
+    "API"
+  ],
+  "npmName": "web3",
+  "npmFileMap": [
+    {
+      "basePath": "dist",
+      "files": [
+        "**/*"
+      ]
+    }
+  ],
+  "authors": [
+    {
+      "name": "Fabian Vogelsteller",
+      "email": "fabian@ethereum.org",
+      "homepage": "https://frozeman.de"
+    },
+    {
+      "name": "Marek Kotewicz",
+      "email": "marek@parity.io",
+      "url": "https://github.com/debris"
+    },
+    {
+      "name": "Marian Oancea",
+      "url": "https://github.com/cubedro"
+    },
+    {
+      "name": "Gav Wood",
+      "email": "g@parity.io",
+      "homepage": "https://gavwood.com"
```

Comment:
Let's use `http://gavwood.com` because https is unsafe link. (this website's SSL certification didn't be authenticated.)
(Please modify the first commit)
Thanks.